sevntu-checkstyle-sonar-plugin/src/main/resources/com/github/sevntu/checkstyle/sonar/checkstyle-extensions.xml

```diff
@@ -377,6 +377,20 @@
     </param>
   </rule>
 
+  <rule>
+    <key>com.github.sevntu.checkstyle.checks.coding.ForbidCertainMethodsCheck</key>
+    <name>Forbid Certain Methods</name>
+    <category name="coding"/>
+    <description>Forbids certain methods in class.</description>
+    <configKey>Checker/TreeWalker/com.github.sevntu.checkstyle.checks.coding.ForbidCertainMethodsCheck</configKey>
+    <param key="methodName" type="REGULAR_EXPRESSION">
+      <description>Method name regexp.</description>
+    </param>
+    <param key="argumentCount" type="STRING">
+      <description>Argument count regexp.</description>
```

Comment:
Regexp is still here, please do search in whole commit yo make sure nothing is left.

Reply:
Done.